### ISSUE TITLE
When using a screen reader the Resolve button in the Merge Conflicts dialog box is not activating

### ISSUE THREAD

**Issue by sb356607** [desktop/desktop#22198]
### The problem

I am having an issue in the Github Merge Branches when there is a conflict I am unable to press spacebar or enter on the Resolve button.  I have also used the screen reader mouse commands with no joy.

### Release version

Version 3.5.11 (x64) (release notes)

### Operating system

Windows 11

### Steps to reproduce the behavior

So when I go to Branches and then over to Merge and most things of course will merge fine but when one runs in to that area where they are thing to resolve the conflicts this is where i have trouble with the swcreen reader.  The fix for now is just to use the command line interface.

**Comment by niik** [comment]
@sb356607 Thank you for the report! I've been unable to reproduce this on macOS and I'm having some issues getting my Windows VM to start just now. As I'm sorting that out to try and reproduce it on Windows are could you please let me know what assistive technology software you're using and what version of it?

**Comment by tidy-dev** [comment]
I have done the following on GitHub Desktop 3.5.12 on 

1. Opened the Branch main menu.
2. Selected "Merge into current branch.."
3. In Merge into {branch name} dialog, I selected a branch that said it would conflict.
4. I hit Create a merge commit.
5. The "Resolve conflicts before Merge" opens. 
6. I navigate to the first conflicted file. I hit enter on the "File resolution options" menu dropdown. 
7. I try to press down arrow to navigate menu options and it moves to next item in the dialog and leaves the list.

If I turn NVDA off, it behaves as expected and navigates the list. 

It does not appear to be an app wide context menu issue. If I use Shift + F10 to open a context menu on a change list item, then press down arrow it navigates the menu while NVDA is on.

Edit: Tested on 3.5.8; I am wondering if this behavior has been present for a long time.

------
Other thoughts: It is unclear if this is the issue that the user is reporting as the wording they are using does not match my reproduction steps.
